src/scores/categorical/contingency_impl.py

```diff
@@ -80,13 +93,63 @@ def __str__(self):
 
     def get_counts(self) -> dict:
         """
-        Return the contingency table counts (tp, fp, tn, fn)
+
+        Returns:
+            dict: A dictionary of the contingency table counts. Values are xr.DataArray instances.
+            The keys are:
+
+            - tp_count for true positive count
+            - tn_count for true negative count,
+            - fp_count for false positive count
+            - fn_count for false negative count
+            - total_count for the total number of events.
+
+        Here is an example of what may be returned:
+
+        .. code-block :: python
+
+            {'tp_count': <xarray.DataArray ()> Size: 8B array(5.),
+             'tn_count': <xarray.DataArray ()> Size: 8B array(11.),
+             'fp_count': <xarray.DataArray ()> Size: 8B array(1.),
+             'fn_count': <xarray.DataArray ()> Size: 8B array(1.),
+             'total_count': <xarray.DataArray ()> Size: 8B array(18.)}
+
         """
         return self.counts
 
     def get_table(self) -> xr.DataArray:
         """
-        Return the contingency table as an xarray object
+        Return:
+            xr.DataArray: The contingency table as an xarray object. Contains
+            a coordinate dimension called 'contingency'. Valid coordinates
+            for this dimenstion are:
+
+            - tp_count for true positive count
+            - tn_count for true negative count
+            - fp_count for false positive count
+            - fn_count for false negative count
```

Comment:
As above, do you need to say "true positive (hit) count" and so forth, or this not necessary in this context?

Reply:
I don't have a strong opinion but I think it's not necessary here.